Python/Python.sublime-syntax

```diff
@@ -15,38 +16,205 @@ file_extensions:
   - Snakefile
 first_line_match: ^#!/.*\bpython\d?\b
 scope: source.python
+
 variables:
   # We support unicode here because Python 3 is the future
   identifier_continue: '[[:alnum:]_]'
   identifier: '\b[[:alpha:]_]{{identifier_continue}}*\b'
-  path: '({{identifier}}\.)*{{identifier}}'
+  path: '({{identifier}} *\. *)*{{identifier}}'
   sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER)\b
 
 contexts:
   main:
     - include: statements
 
   statements:
+    - include: inline_statements
+    - include: block_statements
     - include: classes
     - include: functions
     - include: decorators
+    - include: modifiers
+    - include: assignments
+    - match: ;
+      scope: punctuation.separator.statements.python
```

Comment:
Consistent with what? Other languages all use `punctuation.terminator` for `;`, as far as I know.

Languages using `punctuation.terminator`:

 - C/C++
 - JavaScript
 - CSS
 - Java
 - Objective-C
 - PHP
 - TCL

`punctuation.separator` is used almost exclusively for `,` and `:`.